typescript/lambda-cron/index.ts

```diff
@@ -1,7 +1,7 @@
 import events = require('@aws-cdk/aws-events');
 import targets = require('@aws-cdk/aws-events-targets');
 import lambda = require('@aws-cdk/aws-lambda');
-import cdk = require('@aws-cdk/cdk');
+import cdk = require('@aws-cdk/core');
 
 import fs = require('fs');
 
```

Comment:
cdk still in use here
`export class LambdaCronStack extends cdk.Stack {
  constructor(app: cdk.App, id: string) {
    super(app, id);`

Reply:
My mistake, just noticed you imported core as cdk. Might be worth changing everything to reference 'core.' instead of 'cdk.' to ensure that nothing slips through and to make it clear that we're correctly using the new core library. Given these are examples, I think it's confusing to use 'cdk.' - it's logical to assume that's referencing aws-cdk.cdk if taken as a code snippet.

Reply:
While I don't disagree with you. I think should be something that AWS should dictate from a styling perspective. I can see arguments for both sides. Is there a member from AWS who is committed to this repo that can comment on this?

Reply:
There isn't an owner named on the repo, but the last PR merged (15 days ago) was @NGL321 - probably a good starting point.

Reply:
@NGL321 @eladb any thoughts?

Reply:
For now, I think it is fair to continue to import the core library as "cdk". The content of the library is still the body of the CDK, and "core" was simply a convention change to indicate the _core_ of the CDK.